### ISSUE TITLE
High CPU usage on home/inbox page

### ISSUE THREAD

**Issue by lily-mara** [omnivore-app/omnivore#2821]
While leaving a browser tab idle in the background on the "Home" page, I observe that the Omnivore app consuming 99.9% of available CPU time. This has happened more than once both with an empty home page and with a few articles in my inbox.

I can try to provide more information if this re-ocurrs, what kind of debugging info would be helpful here?

**Comment by jacksonh** [comment]
Hi thanks. I guess as a starting point browser + OS would help. I haven't seen anything similar before, but I wonder if maybe because there are so few items in the library its stuck in an infinite loop trying to refresh.

If you are familiar with the browser Developer tools, opening the network tab to see if requests are being repeatedly made would help.

**Comment by lily-mara** [comment]
Right, I'm on macOS 13.5.2 and using Vivaldi 6.2.3105.51, which is a Chromium derivative. I'll investigate the network requests and JS console the next time this happens.

**Comment by pascalwhoop** [comment]
seeing the same now on Orion browser when creating a "native app" from Omnivore. 
![CleanShot 2024-05-06 at 13 54 48@2x](https://github.com/omnivore-app/omnivore/assets/2284951/ae01afd2-d6c1-41c3-95a8-e395f16f41bc)

![CleanShot 2024-05-06 at 13 55 04@2x](https://github.com/omnivore-app/omnivore/assets/2284951/e3fc4ca3-9699-47c2-a592-bfcde00c5e2c)

**Comment by jacksonh** [comment]
@pascalwhoop are you seeing it make network requests? Also, any chance in the library there are items that are stuck in a pending state (they'd have a green loading type indicator on their thumbnail?

Finally, if you refresh the browser to reload does the issue continue?

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
